public/index.html

```diff
@@ -15,6 +15,8 @@
     <title>Data Ocean</title>
   </head>
   <body>
+    <div id="preloader"></div>
```

Comment:
Preloader should be controlled by `React`

Reply:
rewrited